etcdctl/snapshot/v3_snapshot.go

```diff
@@ -348,41 +366,6 @@ func (s *v3Manager) saveDB() error {
 
 	// db hash is OK, can now modify DB so it can be part of a new cluster
 	db.Close()
-	dbClosed = true
-
-	commit := len(s.cl.Members())
-
-	// update consistentIndex so applies go through on etcdserver despite
-	// having a new raft instance
-	be := backend.NewDefaultBackend(dbpath)
-	defer be.Close()
-
-	ci := cindex.NewConsistentIndex(be.BatchTx())
```

Comment:
Why is setting consistent index removed from this function?

Reply:
Good observation. Trying to develop a good test that would catch such inconsistency...

Reply:
@jingyih: PTAL. I fixed this problem and added a test that would detected it. See `server/verify/verify.go`.